### PR TITLE
Flugzeug hinzufügen update

### PR DESCRIPTION
**Pull request by simon10419**

### CODE REVIEW

File: src/app/plane-list/add-plane-dialog/add-plane-dialog.component.ts [hunk 1]
```diff
@@ -27,19 +27,33 @@ export class AddPlaneDialogComponent {
...
+  pricePerBookedHourFormControl: FormControl;
 
   constructor(public addPlaneDialogRef: MatDialogRef<AddPlaneDialogComponent>, public snackBar: MatSnackBar) {
     this.number = '';
     this.name = '';
     this.possibleAuthorizations = ['PPL-A', 'PPL-B', 'BZF-I', 'BZF-II'];
     this.position = ['Halle 1', 'Halle 2'];
     this.submitted = false;
+    this.pricePerBookedHour = 0;
+    this.pricePerFlightMinute = 0;
```
Comment (Nisatru) [inline src/app/plane-list/add-plane-dialog/add-plane-dialog.component.ts]: Do you ever need this two variables?

File: src/environments/environment.ts [hunk 2]
```diff
@@ -4,7 +4,8 @@
 
 export const environment = {
   production: false,
-  baseUrl: 'https://thomas-cloud.de/uni/api'
+  baseUrl: 'http://localhost:8080'
+  /*baseUrl: 'https://thomas-cloud.de/uni/api'*/
```
Comment (Nisatru) [inline src/environments/environment.ts]: You can delete the old BaseUrl

**Comment by Nisatru** [comment]
Changed target branch from master to integration-test

File: src/app/plane-list/add-plane-dialog/add-plane-dialog.component.ts (hunk 1 above)
Comment (simon10419) [reply src/app/plane-list/add-plane-dialog/add-plane-dialog.component.ts]: Guter Einwand

File: src/environments/environment.ts (hunk 2 above)
Comment (simon10419) [reply src/environments/environment.ts]: Ja nehm ich raus

**Pull request closed without merging**